config/settings.php

```diff
@@ -70,5 +70,6 @@
     'defaults' => [
         'logo' => env('DEFAULT_LOGO', '/images/logo.svg'),
         'room_limit' => env('DEFAULT_ROOM_LIMIT',-1),
+        'own_rooms_pagination_page_size' => env('OWN_ROOMS_PAGINATION_PAGE_SIZE',5),
```

Comment:
Shouldn't there be one equal pagination size for all listings? In this case a more general name would make sense!

Reply:
Since this is not a normal list, but some cards/boxes/buttons, I would prefer a smaller page size than for tabular listings.